tools/lldb-moduleimport-test/lldb-moduleimport-test.cpp

```diff
@@ -169,10 +169,15 @@ collectASTModules(llvm::cl::list<std::string> &InputNames,
           (ELF && Name == swift::ELFASTSectionName) ||
           (COFF && Name == swift::COFFASTSectionName)) {
         uint64_t Size = Section.getSize();
-        StringRef ContentsReference;
-        Section.getContents(ContentsReference);
+
+        llvm::Expected<llvm::StringRef> ContentsReference = Section.getContents();
+        if (!ContentsReference) {
+          llvm::outs() << "error: " << name << " "
```

Comment:
I think that this should be `llvm::errs` for the error message.

Reply:
That sounds right to me, but I'm going to go ahead and merge this because master-next is totally busted right now. Can you address that comment in a follow-up PR?

Reply:
#24977